### PR TITLE
fix: collection of vr fixes

### PR DESCRIPTION
**Pull request by ricardoquesada**
- Node::additionalTransform() and Node::setNodeToParentTransform() can be used at the same time
- Node allocs _additionalTransform in runtime taking 60 bytes lees for 99% of the nodes
- Node calculates the transform faster when an additionalTransform is being used
- VR works when the camera is being moved manually with setNodeToParentTransform
- VR has its own tests
- VR can be disabled and works as expected

### CODE REVIEW

File: web [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit d3f467431111ef39753dfa138d4e6f4b0c70f5c7
+Subproject commit 2b3a2df2cf17552daf2e78fcb69b944412090d26
```
Comment (dumganhar) [inline web]: Does it need to change the reference of the `web` sub module?

Comment (ricardoquesada) [inline web]: nope. thanks. I'll fix it.

**Comment by ricardoquesada** [comment]
@minggo this fixes many bugs in VR... it would be great if this can be part of v3.12.

**Comment by minggo** [comment]
Yep, i just wait the jenkins to pass it.

**Comment by ricardoquesada** [comment]
[ci rebuild]

argh... problems with the slaves..

**Comment by minggo** [comment]
[ci rebuild]

**Comment by minggo** [comment]
And there is conflict.

**Comment by ricardoquesada** [comment]
fixed. thanks

**Comment by minggo** [comment]
No, the conflict still exist.

**Comment by ricardoquesada** [comment]
@minggo yep, you were right. a recent commit on v3.12 created the conflicts. fixed.

**Comment by minggo** [comment]
Thanks, i will merge it when it passes jenkins.

**Pull request merged**